### PR TITLE
Commit theme survey files

### PR DESCRIPTION
**Pull request by bertiewooster**
Theme survey

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,2 @@
...
-
-This repo contains an analysis of the 2021-22 SymPy documentation survey
-results.
-
-The analysis can be seen at https://sympy.org/sympy-docs-survey/
-
-Note, the full data from the survey are not provided here because the survey
-stated that all responses would remain private.
+# sympy-docs-theme-survey
+Analysis of the 2022 SymPy documentation site (docs.sympy.org) theme survey results
```
Comment (asmeurer) [inline README.md]: I'm assuming you didn't mean to revert this.

File: LICENSE [hunk 2]
```diff
@@ -1,29 +1,21 @@
-BSD 3-Clause License
```
Comment (asmeurer) [inline LICENSE]: I don't actually remember using the same license as SymPy here, but I think we should keep it.

**Comment by asmeurer** [comment]
I went ahead and pushed a bunch of changes. I'll need to deploy the site to see if everything works, so I'm going to just merge this and see how it goes.

**Comment by asmeurer** [comment]
OK, there's now a main index page for both surveys https://www.sympy.org/sympy-docs-survey/

**Comment by bertiewooster** [comment]
Great, thanks! Yes, agree with all your changes. Nice looking index page!

I'll go ahead and post about the survey results.

**Pull request merged**